src/conn/conn_layered.c

```diff
@@ -525,19 +525,9 @@ __wt_layered_table_manager_start(WT_SESSION_IMPL *session)
     conn = S2C(session);
```

Comment:
Shouldn't the name of this function now trip `s_all` because it's not used outside this file or module?

Regardless, this can now be made static and renamed `__layered_table_manager_start`.